### DISCUSSION TITLE
Changing footer and License

### DISCUSSION THREAD

**Discussion by mesbahtanvir** [CaiJimmy/hugo-theme-stack#319] [Q&A]
```
Licensed under the GNU General Public License v3.0

Please do not remove the "Theme Stack designed by Jimmy" text and link.

If you want to port this theme to another blogging platform, please let me know🙏.
```

`All rights granted under this License are granted for the term of copyright on the Program and are irrevocable provided the stated conditions are met. This License explicitly affirms your unlimited permission to run the unmodified Program.` GPL v3.0

I am trying to understand the relation between the License and the suggestion.  In my understanding, the GPL v3.0 license does not legally prohibit removing the footer using partials as it's not changing the program anyway. So it is not legally required to get explicit permission to change the footer.

Is my understanding correct? 
If correct, I suggest having other licensing models, which satisfies the author's need. 

I am bringing this because I liked the theme, but I want to remove the footer. Also, I am seeking free options.  

I am just trying out https://www.mesbahtanvir.com/ here

**Comment by CaiJimmy** [comment]
I'm not a lawyer, so what I'm about to say might not be correct: I had understood that, by using a product licensed in GPL-3, it's necessary to give it some kind of credit ("copyright notice"). That's why I added that copyright note in the footer. 

>  If correct, I suggest having other licensing models, which satisfies the author's need.

Thanks for the advice. Do you have any recommendations?

> I am bringing this because I liked the theme, but I want to remove the footer. Also, I am seeking free options.

Sorry. Please keep the copyright note if you want to use this theme.

**Reply by mesbahtanvir** [reply]
Thanks for your response.

> I had understood that, by using a product licensed in GPL-3, it's necessary to give it some kind of credit ("copyright notice"). That's why I added that copyright note in the footer.

The common practice is to add copyright text in the source code as comments, to protect the source code and the program. If the external event can change the behavior then GPL-3 does allow that `unlimited permission to run the unmodified Program`

> Thanks for the advice. Do you have any recommendations?

There might be, but I can't think of any license to satisfy that need.

> Sorry. Please keep the copyright note if you want to use this theme.

Thanks for the suggestion again, will look for other themes, to respect your suggestion.
